### DISCUSSION TITLE
Alert only some cgroups or namespaces in K8s

### DISCUSSION THREAD

**Discussion by Franck31** [netdata/netdata#15973] [Q&A] [answered]
Hi Netdata team,

In our team, we have a large platform with many pods, but I want to receive alerts from specific containers (cgroups) or specific namespaces.  Is there any way to achieve that?

In the documentation, I couldn't find anything in the alerting section related to this.

Cheers!

**Comment by MrZammler** [comment, marked as answer]
Hi @Franck31 

You can define the alerts to work only on specific instances of charts by filtering them with [chart labels](https://learn.netdata.cloud/docs/alerting/health-configuration-reference#alert-line-chart-labels).

For example you could use the chart labels `k8s_container_name` to only enable those alerts for specific namespaces.

**Comment by Franck31** [comment]
Hi @MrZammler 

need a little help from you :)

I try to do what you said, but still not working 
I got a namespace called prod-jf-test, the chart label matches this name 

![image](https://github.com/netdata/netdata/assets/13457574/69156ecc-81b8-4642-ad0c-248e3991c48e)

But when i modified my cgroup cfg to use chart labels to not alert this namespace is not working

```
    cgroups:
      enabled: true
      path: /etc/netdata/health.d/cgroups.conf
      data: |
       template: k8s_cgroup_ram_in_use
              on: k8s.cgroup.mem_usage
           class: Utilization
            type: Cgroups
       component: Memory
              os: linux
           hosts: *
            calc: ($ram) * 100 / $memory_limit
           units: %
           every: 10s
            warn: $this > (($status >= $WARNING)  ? (80) : (91))
            crit: $this > (($status == $CRITICAL) ? (91) : (98))
       chart labels: k8s_namespace=!prod-jf-test *
           delay: down 15m multiplier 1.5 max 1h
            info: cgroup memory utilization
              to: sysadmin

```
and the alert stills alive 

![image](https://github.com/netdata/netdata/assets/13457574/e37d9b15-1886-4ade-ae47-2168403ae9f6)


BTW im using netdata 1.38.0

Any ideas?

Cheers!

**Reply by Franck31** [reply]
I fixed myself, it was a version issue, the filter using chart labels is enabled in Netdata > 1.40.0 [#15086](https://github.com/netdata/netdata/pull/14982)

Also i think there is a typo in the docs: https://learn.netdata.cloud/docs/alerting/health-configuration-reference#alert-line-chart-labels, at the end of every line there is a "`" maybe can confuse some people. 

Cheers!

**Reply by MrZammler** [reply]
Thanks! Will fix the doc, thank you!
